### DISCUSSION TITLE
Release lists contains wide empty content

### DISCUSSION THREAD

**Discussion by Karlatemp** [community/community#6041] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/30070303/136232545-50d0dd7f-2fde-4e04-8cf4-71e4218c6587.png)
How to put it, it looks very strange, with so wide empty.

It looks even stranger at higher resolutions.

![image](https://user-images.githubusercontent.com/30070303/136234127-0983b184-180b-4c81-bf52-15470888f86c.png)

**Comment by wrooot** [comment]
Yes. hate this new design idea to make everything bulkier and spacier. Instead of mobile versions of sites now everything is tailored for a touch input.

**Comment by MylesBorins** [comment]
I've shared this feedback with our designer to see if there are any tweaks we can make here. That said we have had multiple discussions about the use of whitespace and this design is quite intentional, even if not to your preference.

**Comment by Soundtoxin** [comment]
This was my first thought upon enabling this feature preview. "wow, that's a lot of wasted space". I had to carefully read the existing feedback entries to find this complaint, but I was feeling sure someone else would've thought the same.

**Comment by rochacon** [comment]
Adding to this, I also think the left column whitespace disconnects its content from the release itself. Just aligning it to the right again (like in previous design), close do the release details, would be better. Or, just get rid of the left column and use the new space inside the release details for this info.

**Comment by MylesBorins** [comment, marked as answer]
As an update we've tightened the space on the left column, PTAL
